schema/custom-codec-schema.json

```diff
@@ -62,6 +62,7 @@
             "RingbufferStoreConfigHolder",
             "TimedExpiryPolicyFactoryConfig",
             "DurationConfig",
+            "ClientBwListEntry",
```

Comment:
Why not define `EntryList_int_String` which will be more reusable for other codecs if needed rather than making it MC specific?

Reply:
I have considered this option. I decided to use a custom codec instead of introducing the `EntryList_int_String` one as there is a pretty good chance that we'll need to add more fields into `ClientBwListEntry` type in the future.

Reply:
@ihsandemir Does it make sense to you? Or you still think it's worth it to introduce `EntryList_int_String` custom type?